docs/faq_tips_tricks.rst

```diff
@@ -0,0 +1,35 @@
+.. include:: links.rst
+
+========================
+FAQ, Tips, and Tricks
+========================
+
+Should I run quality control of my images before running fMRIPrep?
+--------------------------------------------------------------------
+
+Yes. You should as well before any other processing/analysis.
+
+Oftentimes (more often than we would like), images have fatal artifacts and problems. 
+Those would not pass any QC check.
+Some other times, QC checks will flag some images that we should carefully track throughout processing.
+
+When using publicly available datasets, an additional concern is that images may have gone through some kind of preprocessing (see next question).
+
+
+What if I find some images have undergone some pre-processing already (e.g., my T1w image is already skull-stripped)?
+---------------------------------------------------------------------------------------------------------------------------
+
+These images imply an unknown level of preprocessing (e.g. was it already bias-field corrected?), which makes it difficult to decide on best-practices for further processing.
+Hence, supporting such images was considered very low priority for fMRIPrep.
+For example, see `#707 <https://github.com/poldracklab/smriprep/issues/12>`_ and an illustration of downstream consequences in `#939 <https://github.com/poldracklab/fmriprep/issues/939>`_).
+
+So for OpenFMRI, we've been excluding these subjects, and for user-supplied data, we would recommend reverting to the original, defaced, T1w images to ensure more uniform preprocessing.
+
+
+My fMRIPrep run is hanging...
+---------------------------------
+
+There is a Python bug that affects fMRIPrep when processes are killed for running out of memory. 
```

Comment:
```suggestion
When running on Linux platforms (or containerized environments, because they are built around Ubuntu),
there is a Python bug that affects fMRIPrep that drives the Linux kernel to kill processes as a response to
running out of memory.
Depending on the process killed by the kernel, fMRIPrep may crash with a ``BrokenProcessPool`` error
or hang indefinitely, depending on settings. 
```